docs/renarde/JavaSupport.md

```diff
@@ -0,0 +1,14 @@
+# Java Support
+
+### REST Endpoints
+
+For Renarde `Controller` classes, Renarde uses a different strategy than JAX-RS to determine which methods count as REST endpoints, and what the path of each endpoint is.
+vscode-quarkus contains functionality so that the existing JAX-RS features (code lens and workspace symbols) are handled according to these rules in `Controller` classes.
+
+![](./images/renarde-code-lens.png)
+
+*JAX-RS code lens in a Renarde application*
+
+![](./images/renarde-workspace-symbols.png)
```

Comment:
To be consistent with Qute documentation, please use white Theme. I think it would be nice to have a little demo which show in action the workspace symbol.

You can use this demo if you wish:

![RenardeWSAndCodelensDemo](https://user-images.githubusercontent.com/1932211/221781646-c36a430d-4719-4336-bebe-9476f006ef4e.gif)

Reply:
Alright, I used this GIF, and I updated the CodeLens image to use the Light theme.